### PR TITLE
Feature/average runtime float

### PR DESCRIPTION
**Pull request by Daniel-ltw**
Address problems in #105

### CODE REVIEW

File: lib/sidekiq/statistic/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Sidekiq
   module Statistic
-    VERSION = '1.2.0'
+    VERSION = '1.2.1'
```
Comment (davydovanton) [inline lib/sidekiq/statistic/version.rb]: hey, could you revert this changes? I'll do it by self

**Comment by Daniel-ltw** [comment]
@davydovanton done

**Comment by davydovanton** [comment]
thanks! I'll merge this after fixing s3 issue (CI doesn't work 😞 )

**Comment by Daniel-ltw** [comment]
Yeah, sad. Internet is down with all this AWS drama.

**Comment by davydovanton** [comment]
awesome work, thanks 👍

**Pull request merged**